notebooks/contributing_index.md

```diff
@@ -0,0 +1,40 @@
+Contributing to User Tutorials
+==============================
+
+SimPEG welcomes contributions by the community to improve available tutorial materials.
+However, tutorials must adhere to specific guidelines in order to ensure a high level of quality and consistency.
+For those interested in contributing to SimPEG User Tutorials, please follow the instructions provided.
+
+
+Before Creating a New Tutorial
+------------------------------
+
+```{admonition} WARNING!
+:class: danger
+Please ensure that the contents of your tutorial **DOES NOT** fall into any of the following categories:
+
+* the tutorial focusses on functionality that is not part of SimPEG
+* the amount of SimPEG functionality NOT already covered in another tutorial is insufficient
+* the tutorial focusses on a data-specific result and is not generalizable
+* the tutorial was created for the purpose of self promotion
+
+Tutorials that fall into any of these categories will not be accepted!
```

Comment:
I think this is a strong statement that could very likely prevent contributors from opening a PR or wanting to contribute.

What if we add a statement like this one instead:

> In case you have an idea, but are unsure if it satisfies or not these criteria, feel free to open an Issue describing it in detail we can discuss it.

(Feel free to suggest changes to that one, it's just an example).